src/App.jsx

```diff
@@ -89,48 +44,51 @@ function App() {
 		const params = useParams();
 
 		const courseItem = getCurrentCourse(params.courseId);
-
 		const courseAuthors = getCurrentAuthors(courseItem[0].authors);
 
-		return (
-			<CourseInfo
-				courseItem={courseItem}
-				courseAuthors={courseAuthors}
-				userName={userName}
-			/>
-		);
+		return <CourseInfo courseItem={courseItem} courseAuthors={courseAuthors} />;
+	}
+
+	function GoToUpdateCourse() {
+		const params = useParams();
+
+		const courseItem = getCurrentCourse(params.courseId);
+
+		return <CourseForm updateMode='true' course={courseItem} />;
+	}
+
+	function UserMe() {
+		let userInfo = useSelector((state) => state.users);
+		const token = localStorage.getItem('courseUserToken');
+		if (token) {
+			loadUserMe(token)(dispatch);
```

Comment:
this works fine but please make sure to take a look at how thunk actions are called conventionally https://redux.js.org/usage/writing-logic-thunks